website/public/locales/ar/side_menu.json

```diff
@@ -1,12 +1,12 @@
 {
-  "dashboard": "لوحة التحكم",
-  "dashboard_home": "الصفحة الرئيسية للإحصائيات",
-  "leaderboard": "جدول الأوائل",
-  "messages": "رسائل",
-  "messages_dashboard": "لوحة تحكم الرسائل",
+  "dashboard": "لوحة المعلومات",
```

Comment:
We had a bit PR that removed this file. Can you move all these entries into `common.json`?

Reply:
So i should move all the `side_menu.json` entries into `common.json`